benchmarks/bench_sparse_feat_expansion.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python
+
+import matplotlib.pyplot as plt
```

Comment:
I wouldn't have thought we needed a new benchmark for this. The speed gains are obvious to reason about, and there would be little benefit in trying to fine tune it

Reply:
👍 I was mostly trying to substantiate my claim that the time to `fit_transform` is dominated by `fit` on certain data. But, I got the pictures and you seem to agree so... Removed.